src/vm/eetwain.cpp

```diff
@@ -5169,7 +5169,11 @@ OBJECTREF EECodeManager::GetInstance( PREGDISPLAY    pContext,
     if (info.ebpFrame)
     {
         _ASSERTE(stackDepth == 0);
+#if defined(_TARGET_X86_) && defined(FEATURE_PAL)
+        taArgBase = GetCallerSp(pContext) - 2 * sizeof(TADDR);
```

Comment:
This code is already in ``_TARGET_X86_`` region, and thus ``_TARGET_X86_`` is unnecessary.

I think that it would be better to use ``WIN64EXCEPTIONS`` instead of ``FEATURE_PAL``.